diff --git a/node_modules/@fastify/helmet/types/index.d.ts b/node_modules/@fastify/helmet/types/index.d.ts
index f3f788d..0be05e8 100644
--- a/node_modules/@fastify/helmet/types/index.d.ts
+++ b/node_modules/@fastify/helmet/types/index.d.ts
@@ -27,10 +27,10 @@ declare namespace fastifyHelmet {
     helmet?: Omit<FastifyHelmetOptions, 'global'> | false;
   }
 
-  export interface FastifyHelmetOptions extends NonNullable<HelmetOptions> {
+  export type FastifyHelmetOptions = {
     enableCSPNonces?: boolean,
     global?: boolean;
-  }
+  } & NonNullable<HelmetOptions>;
 
   export const fastifyHelmet: FastifyHelmet
   export { fastifyHelmet as default }
